projects/demo-playwright/tests/core/text-area/text-area.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Textarea`, () => {
+    [`m`, `l`].forEach(size => {
+        test(`size of ${size}`, async ({page}) => {
+            await tuiGoto(
+                page,
+                `components/textarea/API?tuiMode=null&tuiTextfieldSize=${size}`,
+            );
+            const example = page.locator(`#demo-content`);
+
+            await expect(example).toHaveScreenshot(
+                `textarea-tuiTextfieldSize-${size}.png`,
```

Comment:
The generated screenshot is too wide

<img width="1166" alt="Screenshot 2023-11-20 at 13 33 10" src="https://github.com/taiga-family/taiga-ui/assets/35179038/d0587afb-8566-4292-aa80-8de7e9d54800">

Reply:
Addressed